inc/ticket.class.php

```diff
@@ -304,14 +306,35 @@ static function addHistoryOnAddGroup(CommonDBTM $item) {
          $group->getFromDB($groups_id);
 
          $task = new TicketTask();
+          $content = __("Escalation to the group", "escalade") . " " . $group->getName();
```

Comment:
If I understand correctly, this code is executed when someone add a new group to the ticket.
This mean that `$item->input['escalade_comment']` will never be set and the condition will never be true (thus not changing the original behavior).

Am I missing something ?

Reply:
When taskhistory is false, we just keep the user's comment when changing groups. Otherwise, we add the information of the escalation with the name of the new group. 
The operation without going through the form remains the same :
taskhistory=true --> escalation added in the timeline 
taskhistory = false --> no task added in the timeline